submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/repositories/data/SubmissionRepository.kt

```diff
@@ -32,11 +35,11 @@ open class SubmissionRepository(
     open fun getSimpleByAccNo(accNo: String): Submission = getExtByAccNo(accNo).toSimpleSubmission()
 
     @Transactional(readOnly = true)
-    open fun getExtByAccNo(accNo: String) = submissionMapper.toExtSubmission(lodSubmission(accNo))
+    open fun getExtByAccNo(accNo: String) = submissionMapper.toExtSubmission(dbToExtRequest(accNo))
 
     @Transactional(readOnly = true)
     open fun getExtByAccNoAndVersion(accNo: String, version: Int) =
-        submissionMapper.toExtSubmission(lodSubmission(accNo, version))
+        submissionMapper.toExtSubmission(dbToExtRequest(accNo, version))
```

Comment:
I understand the need for a request object when the method has a gazillion parameters but for cases like this, I think it's not really necessary. I'm not sure if this is part of a good practice design that I'm not aware of but, just leaving my two cents here.

Reply:
as latter we will include project this is a preparation for that